clusters/sqnc-staging/demanda/attachment-api/values.yaml

```diff
@@ -16,10 +16,12 @@ auth:
   oauth2Realm: demanda
   internalRealm: demanda-internal
   internalClientId: sqnc
+  externalRealm: demanda-external
 ingress:
   enabled: true
   ingressClassName: nginx-demanda
   hostname: demanda.sqnc-stage.com
+authzWebhook: http://demanda-matchmaker-api-sqnc-matchmaker-api.demanda.svc.cluster.local:3000/v1/authz
```

Comment:
Should this exist publicly or privately?

Given eventually these will be externally available, should there be an endpoint configured fot this webhook in the matchmaker helm chart?